desc/compute/geom_utils.py

```diff
@@ -161,3 +161,12 @@ def inner(vec, phi):
         return cart
 
     return inner(vec, phi)
+
+
+def copy_rpz_periods(rpz, NFP):
```

Comment:
when #1555 is in, should unify this utility and the one of the same name from that PR (since they are the same I think?)